.github/workflows/all.yml

```diff
@@ -305,3 +305,18 @@ jobs:
         run: |
           mv tools/semgrep/.semgrepignore . & \
           semgrep --verbose --config="r2c" --config="tools/semgrep/rules/example.yaml"
+
+  lint-cargo-toml:
+    name: lint-cargo-toml
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
Let's pin the version of `actions/checkout` 
actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab